lib/docs/scrapers/vue.rb

```diff
@@ -15,12 +15,12 @@ class Vue < UrlScraper
     options[:replace_paths] = { 'guide/' => 'guide/index.html' }
 
     options[:attribution] = <<-HTML
-      &copy; 2013&ndash;2018 Evan You, Vue.js contributors<br>
+      &copy; 2013&ndash;2019 Evan You, Vue.js contributors<br>
```

Comment:
When you update `options[:attribution]`, please also update the `assets/javascripts/templates/pages/about_tmpl.coffee` with correct licensing information for this documentation.

Through this review I also want to let you that although your contribution is accepted now since we have lots of outdated docs while we're still working on merging old PR's, once I'm done with the "Add x documentation" PR's I'm going to more strictly follow our [contributing instructions on updating docs](https://github.com/freeCodeCamp/devdocs/blob/master/.github/CONTRIBUTING.md#updating-existing-documentations) (so PR's which only update version numbers and/or attribution will no longer be considered after that).

Reply:
First sorry for missing the latest `CONTRIBUTING.md`
So afterwards version upgrade will go through what forms ?

Reply:
This PR (and PR's created before the PR's adding new docs are merged or closed) will still be reviewed and merged, even if they only update version numbers.

Just to clarify a bit: The reason for this "rule" being in place is that updating docs is often about more than just upping the version number and that often the filters need to be changed too. These required changes are often not spotted by people with less experience with the project (not implying you directly), which is why updating documentations is mainly a task for maintainers.

Afterwards version upgrades will yet again be mainly a task for [maintainers](https://github.com/freeCodeCamp/devdocs/blob/master/docs/maintainers.md#list-of-maintainers). I already have plans to set up a cron job that notifies me (and possibly other maintainers) of outdated docs daily to ensure docs won't be outdated for as long as they are now anymore.